packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -252,8 +300,56 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
     }
     return currentPosition;
   }
+
+  // returns an array of spans each span pertains to a custom label the user passes in
+  private _addLabels(cssRegularLabelClassNames: string | undefined, marks: ISliderMarks[]): JSX.Element[] {
+    const { vertical, min, max } = this.props;
+    const labels: JSX.Element[] = [];
+    if (min === undefined || max === undefined || marks === undefined) {
+      return labels;
+    }
+
+    for (let i = 0; i < marks.length; i++) {
+      const currentLabel = (
+        <span
+          className={cssRegularLabelClassNames}
+          style={this._getStyleUsingOffsetPercent(vertical, ((marks[i].value - min) / (max - min)) * 100)}
+          key={i}
+        >
+          {marks[i].label}
+        </span>
+      );
+      labels.push(currentLabel);
+    }
+    return labels;
+  }
+
+  private _addTickmarks(cssRegularTickClassNames: string | undefined): JSX.Element[] {
+    const { min, max, step, vertical } = this.props;
+    // if anyof the values are not defined then will not render ticks
```

Comment:
```suggestion
    // If any of the values is undefined, then we don't render ticks
```

Reply:
And one of my comments about adding a comment like this was meant to add them in the description of the prop were users that read the documentation can see it. In here it's just source code that only us are seeing it.